setup.py

```diff
@@ -16,7 +16,7 @@
       install_requires=[
           'PyGithub',
           'PyYAML',
-          'dateutils',
```

Comment:
Opss, that must have happened while the merge issues I had while rebasing stuff.
I will re-add that.